internal/pkg/secret/encryption_engine.go

```diff
@@ -103,20 +103,7 @@ func (c *EncryptEngineImpl) UnwrapDataKey(dataKey string, iv string, algo string
 	return base64.StdEncoding.DecodeString(dataKeyEnc)
 }
 
-func (c *EncryptEngineImpl) Encrypt(plainText string, key []byte) (data string, ivStr string, err error) {
-	defer func() {
```

Comment:
This doesn't look like a linter change; was removing this block intended?

Reply:
If you think about it, it does nothing, right? I suppose it might catch a `panic()` but it doesn't look like there's anything that could `panic()`...

Reply:
Hey @tadsul, looks like you wrote this code. Do you remember why you chose to use `recover()` here? Were we really seeing a `panic()`?

Reply:
Hmm... I wonder if since `err` is defined in the return type, it's being returned from inside the `defer()`...

Reply:
Nope, this doesn't work: https://play.golang.com/p/B0sKK_uwiWn

Reply:
Yeah sounds good. I tried looking into the history to see if there was some motivation for it and there was a comment asking about tests for those errors but no response.